Makefile.am

```diff
@@ -1,5 +1,3 @@
-ACLOCAL_AMFLAGS = -I m4
```

Comment:
Why do you need this change?

Reply:
When I run some of the autotools it complains that there is no `m4` folder to include. And in fact there isn't, so to prevent it from falling over I remove it.